csm/conf/configure.py

```diff
@@ -150,7 +151,7 @@ def _set_s3_endpoints():
             s3_endpoints_count = len(s3_endpoints)
             for endpoint_count in range(s3_endpoints_count):
                 Conf.set(const.CSM_GLOBAL_INDEX,
-                        f'{const.RGW_S3_ENDPOINTS}[{endpoint_count}]',
```

Comment:
can you please check if const.RGW_S3_ENDPOINTS is getting used ? if not remove it.
Please check if the const key is required for each key you modified.

Reply:
Re,oved from const